tools/networkmonitor/networkmonitor.nim

```diff
@@ -193,7 +193,33 @@ proc crawlNetwork(node: WakuNode,
     # we dont run ipMajorityLoop
     # we dont run revalidateLoop
 
-    await sleepAsync(crawlInterval)
+    await sleepAsync(crawlInterval.millis)
+
+proc retrieveDynamicBootstrapNodes(dnsDiscovery: bool, dnsDiscoveryUrl: string, dnsDiscoveryNameServers: seq[ValidIpAddress]): Result[seq[RemotePeerInfo], string] =
```

Comment:
maybe a comment here that this is the same as the proc in wakunode2?

Reply:
This method probably will change in `wakunode2` app after the refactoring feature branch is merged into the `master` branch. So... I don't think a comment will add any value 🤷🏼